modules/sdk/gradle-plugins-app-javadoc-builder/README.markdown

```diff
@@ -0,0 +1,59 @@
+# App Javadoc Builder Gradle Plugin
+
+The App Javadoc Builder Gradle plugin allows you to generate a combined HTML API
+documentation for an application that spans different subprojects, each one
```

Comment:
@Ithildir I'm not sure what a "combined HTML API documentation" is. Do you mean just generated API docs viewable as HTML? Could we say something like

> ...allows you to generate API documentation as HTML for an application...